src/screens/TestCases/Modal/CustomFilterModal.js

```diff
@@ -9,17 +11,26 @@ const CustomFilterModal = ({
   position,
   filterColumn,
   handleCheckboxChange,
+  handleBetweenValueChange,
   selectedFilters,
   handleSelectAll,
   uniqueValues,
   searchTerm,
-  setSearchTerm
+  setSearchTerm,
+  paginationData,
+  filterColumnId,
+  setFilterType,
+  setFilterText,
+  filterType,
+  columnName
 }) => {
   const [showTooltip, setShowTooltip] = useState(false);
   const [showModal, setShowModal] = useState(false);
   const target = useRef(null);
   const searchRef = useRef(null);
   const containerRef = useRef(null);
+  // const [filterType, setFilterType] = useState('');
```

Comment:
remove all commented code, if not req. else give proper comment

Reply:
resolved